monai/deploy/operators/dicom_data_loader_operator.py

```diff
@@ -90,7 +91,10 @@ def _load_data(self, files: List[str]):
         sop_instances = []
 
         for file in files:
-            sop_instances.append(dcmread(file))
+            try:
```

Comment:
It is good that we catch and suppress the exception on reading non-DICOM part10 files now, though this creates a chance of a silent failure when the given folder contains all non-DICOM files, for our use cases.
Consumers of this operator often check the loaded study list exists and not empty, and here is an [example](https://github.com/Project-MONAI/monai-deploy-app-sdk/blob/64b40b71162eeb759b80f8a644c623133e85aec7/monai/deploy/operators/dicom_series_to_volume_operator.py#L46), but it would be better to raise an error in the compute method of this operator when nothing is loaded.

Reply:
Raise errors would just break unless there is a way to catch on the operator's level and be able to continue to a downstream operator.

Reply:
That's exactly the point, error out at the point of not being able to load the necessary data, in this case, no DICOM study loaded, instead of letting the downstream consumers to raise error on missing required data,